lib/osso-abook-contact-field.c

```diff
@@ -4377,6 +4377,28 @@ contact_action_url(GtkWindow *parent, const char *url)
   return TRUE;
 }
 
+static gboolean
+contact_action_tel_as_uri(GtkWindow *parent, const char *line_id)
+{
+  gchar *uri = g_strconcat("tel://", line_id, NULL);
+  HildonURIAction *action = hildon_uri_get_xdg_action();
```

Comment:
I am not sure we shall prefer xdg-open  here, lets leave libhidonmime decide for us